### PR TITLE
Feature/sequencing and metadata coverage

### LINKED ISSUE #154: Sequencing representativeness in Switzerland
**Aim**

I'm thinking about adding a deep focus page to the sequencing intensity box for Switzerland. The sequencing intensity plot shows us how consistent we sequence through time. The aim of the new widget is to help us analyze whether we sample representatively with respect to the cantons, age, and sex.

It should allow some slightly deeper analysis and, for example, not only answer the question "How representative did we sample w.r.t. the cantons?" but also "How representative did the cantons sample w.r.t. different age groups?"

**Idea**

An idea is to have the attributes division, age, and sex as checkboxes and the user can choose only one...

image

or multiple:

image

This will of course create a lot of bars but I think that it is still easy to understand and to use. If we allow all three to be checked, it would create about 500 rows.

**API**

(The required endpoints are not there yet but we have the data and can create them once we decided how the plot should look like.)

(@TKGZ, what do you think?)

### LINKED ISSUE #162: Metadata completeness
Before viewing and "trusting" the plots that largely depend on the availability of metadata (e.g., the age and geographic distribution plots and hospitalization probabilities), it is important to know how complete the dataset is. Thus, we should have a plot showing the proportion of sequences for which we know the division, age, sex, hospitalization, and death status.

### PR DESCRIPTION
**Pull request by chaoran-chen**
This PR adds a "deep" page which is linked from the "show more" button of the sequencing intensity plot. Because the `DeepFocusPage` is variant-specific, I introduced the `DeepExplorePage`.

The aim of the new `SequencingCoverageDeepExplore`-page is to give an overview of the quality/completeness/representativeness of the data. It contains the `MetadataAvailabilityPlot` which is available for all countries (closes #162) and the `SequencingRepresentativenessPlot` which is only available for Switzerland (closes #154).

image

### CODE REVIEW

**Comment by spacephoton** [comment]
Do you think we can make it without scrollbar and have it extend to the bottom instead?

CleanShot 2021-06-07 at 18 43 53@2x

**Comment by chaoran-chen** [comment]
I think that it is nice that the left and right parts don't move. It becomes more important if the user selects both division and age group, then the plot gets really long.

File: src/pages/DeepExplorePage.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        samplingStrategy={props.samplingStrategy}
+        sequencingIntensityEntrySet={props.sequencingIntensityEntrySet}
+      />
+    ),
+  },
+];
+
+export const DeepExplorePage = (props: Props) => {
+  const { path, url } = useRouteMatch();
+  const _makeLayout = (content: JSX.Element) =>
```
Comment (spacephoton) [inline src/pages/DeepExplorePage.tsx:35]: why does this have an undersscore?

**Review by spacephoton — APPROVED** [review]
Ok, makes sense, since some lists are very long/

**Comment by spacephoton** [comment]
Seems great!! :D

**Comment by spacephoton** [comment]
Maybe a subtitle explaining a little bit more here?
CleanShot 2021-06-07 at 18 49 44@2x

**Comment by spacephoton** [comment]
Because list is sometimes long, maybe we can put x axis above?

**Comment by spacephoton** [comment]
Maybe useful to also sort by cases?

**Comment by spacephoton** [comment]
Or have that option

File: src/pages/DeepExplorePage.tsx (hunk 1 above)
Comment (chaoran-chen) [reply src/pages/DeepExplorePage.tsx:35]: Because it is importing a function called `makeLayout` from `'../helpers/deep-page'` (line 3)

**Comment by chaoran-chen** [comment]
> Maybe a subtitle explaining a little bit more here?

Done

> Because list is sometimes long, maybe we can put x axis above?
> Maybe useful to also sort by cases? Or have that option

Good ideas. Let's address them in future PRs.

**Comment by spacephoton** [comment]
Awesome!

**Pull request merged**